imagetest/test_suites/metadata/shutdown_script_test.go

```diff
@@ -0,0 +1,60 @@
+
+package metadata
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+const shutdownTime = 110 // about 2 minutes
+
+// TestGuestShutdownScript test that shutdown scripts can run for around two minutes
+func TestGuestShutdownScript(t *testing.T) {
```

Comment:
rename to TestShutdownScriptTime, since that's what's different from TestShutdownScript
```suggestion
func TestShutdownScriptTime(t *testing.T) {
```